src/bin/pgcopydb/cli_common.c

```diff
@@ -825,9 +854,11 @@ cli_copy_db_getopts(int argc, char **argv)
 		exit(EXIT_CODE_BAD_ARGS);
 	}
 
+	const char *optstring =
+		"S:T:D:J:I:b:L:cAPOXj:xBeMlUdyF:F:Q:irRCN:fp:ws:o:tE:Vvdzqh";
```

Comment:
I do not see `g` character on this line. You appear to add the characters `d` and `y` in this PR.

If you try the short version of the new argument, you get `clone: invalid option -- 'g'`.